pkgs/tools/networking/network-manager/default.nix

```diff
@@ -14,7 +14,9 @@ stdenv.mkDerivation rec {
   };
 
   preConfigure = ''
-    substituteInPlace tools/glib-mkenums --replace /usr/bin/perl ${perl}/bin/perl
+    substituteInPlace configure --replace /usr/bin/uname ${coreutils}/bin/uname
+    substituteInPlace configure --replace /usr/bin/file ${file}/bin/file
```

Comment:
If they have `configure.ac` it may be better to add `autoreconfHook` to `buildInputs` instead.